deployment/Heroku/ci/GitHub/.github/workflows/heroku_deploy.yml.j2

```diff
@@ -35,7 +35,9 @@ jobs:
             heroku_email: {{ '${{ secrets.HEROKU_EMAIL }}' }}
             region: {{ctx.current_template['parameters']['HEROKU_REGION']}}
             usedocker: true
+            {% if ctx.templates|length > 1 %}
```

Comment:
Does this context.templates contain the additional ci templates or deployment templates? Or just code templates?

Reply:
You are absolutely right. I realised the same thing last night. Will fix.